browser/help/topics/dataset-selection.md

```diff
@@ -7,17 +7,21 @@ The data selection drop-down specifies which dataset is displayed on the page. I
 
 In the past, the gnomAD browser has supported disease-specific subsets (e.g. non-cancer). However, **we have decided to no longer create these subsets in v4 and will begin to phase out support for all subsets in the browser in 2024**. We made the decision to remove support for subsets from the gnomAD v4 exomes for two reasons: sample metadata and sample size. While we are provided high level study phenotype and case/control status for some samples, **we do not have comprehensive phenotype metadata for gnomAD samples**, and many samples are now derived from large biobanks, which can include individuals with disease. As such, we cannot ensure that samples in a non-disease subset do not have the specified disease. Additionally, as the dataset has grown, concerns about enrichment of any particular phenotype decreases. And finally, we continue to remove cohorts recruited for severe pediatric disease and have also removed the TCGA cancer samples due to data quality. For a more detailed understanding of the cohorts included in gnomAD, please see the “Study Diseases in gnomAD” table [on our stats page](/stats#study-diseases-in-gnomad). In summary, we encourage our users to check frequencies across the full dataset to fully leverage the scale and power of gnomAD data.
 
-Note that gnomAD v4.0 has a single subset (non-UK Biobank), but frequencies for this subset are not displayed on the browser and are only available in our downloadable files.
+Note that gnomAD v4.1 has a single subset (non-UK Biobank), but frequencies for this subset are not displayed on the browser and are only available in our downloadable files.
```

Comment:
How about here if they will be available in Browser? @ch-kr

Reply:
we could add a note, but we don't have any notes for other displayed subsets, e.g.
> gnomAD v3.1.2 (non-cancer): Only samples from individuals who were not ascertained for having cancer in a cancer study.

though I'll will add a note now to make this clearer this only impacts the exomes

Reply:
oh I can't read, will update this also